spade-aws/src/main/scala/com/salesforce/mce/spade/aws/SpadeAwsContext.scala

```diff
@@ -34,7 +40,7 @@ object SpadeAwsContext {
       Emr(
         config.getString("release-label"),
         config.getString("subnet-id"),
-        config.getString("ec2-keyname"),
+        Try(config.getString("ec2-keyname")).toOption,
```

Comment:
let's use `optionIfMissing` in `SpadeContext` to make this only convert to Option when missing